packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2385,6 +2394,47 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         }
     }
 
+    private submitMaybeChunkedMessages(
+        type: ContainerMessageType,
+        content: any,
+        serializedContent: string,
+        serverMaxOpSize: number,
```

Comment:
(nit) I feel like logically, at this level, this._maxOpSizeInBytes & serverMaxOpSize are one and the same. The only difference if chunking is enabled or not.

So maybe it's easier to represent that state not as triplet (_maxOpSizeInBytes, serverMaxOpSize, _chunkingDisabled), but as a tuple:


const maxOpSize = !this._chunkingDisabled ? this.context.deltaManager.maxMessageSize : (this.mc.config.getNumber(maxOpSizeInBytesKey) ?? defaultMaxOpSizeInBytes)